pages/faq.md

```diff
@@ -0,0 +1,103 @@
+
+##### How is AB Tech different from technical theatre or Scotch ‘n’ Soda?
+While we do partner and share equipment with the drama department and Scotch ‘n’ Soda (in fact, some of our members are also involved with S’n’S), as the university’s production organization, we support a wider range of events. We specifically specialize in event production with handling entertainment technology; however, you can expect some overlap in knowledge between us and S’n’S.
+
+<h2>
+ <a id="Getting Involved">Getting Involved</a>
+</h2>
+##### Why should I join AB Tech?
+To play with some of the most expensive equipment CMU owns, of course! Our lighting boards, sound systems, and supporting equipment are all what you would find in the industry, and we’ll teach you how to use them so you can design the next big event. Plus, you’ll get to be involved with some of the most exciting productions of the year. If you’re interested in working with lights, speakers or being backstage, come meet fellow techies and join our next event!
+
+##### What can I do as a techie?
+Techies can take on several different run crew positions, which are roles that are responsible for specific aspects of an event (for example, lighting). Techies can also help with general setup and strike. Some common run crew positions include:
+<ul>
+ <li>Techie-in-Charge (TiC): Responsible for planning the event, including communicating with the event organizer, finding run crew and coordinating the logistics of the event.</li>
+ <li>Lighting Designer (LD): Responsible for designing where lighting fixtures are placed around the venue and programming the lighting.</li>
+ <li>Front of House Engineer (FoH): Responsible for selecting appropriate speaker systems and mixing inputs so that the show sounds good to the audience.</li>
+ <li>Stage Manager (SM): Responsible for communicating between performers backstage and run crew in house, as well as calling cues and handing off equipment.</li>
+ <li>Assistant Postions: These are assistant positions that are great for beginners to learn about a run crew position. Especially for large events, you may see aLD, aFoH and aTiC positions open where you can work side by side with a more experienced run crew member.</li>
+</ul>
+
+##### Do I need prior experience in event production?
+No, in fact, many of our members have little to no prior experience in event production when starting out. Just about everything can be learned on the job by participating in events and taking run crew positions. 
+
+##### When does AB Tech meet?
+AB Tech holds open weekly meetings for anyone interested in getting involved with upcoming events. Join our mailing list in the <a href="/join">Join</a> form for specific times and locations. We also meet to set up events throughout the week. For a list of upcoming events, please see our <a href="https://abtech.andrew.cmu.edu/tracker/" target="_blank">Tracker</a> site. 
```

Comment:
Change links to be `<a href="{{ '/join/' | relative_url }}">` and `<a href="https://tracker.abtech.org/" target="_blank">`.